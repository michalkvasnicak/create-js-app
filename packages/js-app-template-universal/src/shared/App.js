/* @flow */
import 'normalize.css';
import Helmet from 'react-helmet';
import React from 'react';
import styles from './App.css';

export default function App(/* props: Object */) {
  return (
    <div>
      <Helmet title="Hello World" />
      <h1 className={styles.red}>Hello World</h1>
    </div>
  );
}
