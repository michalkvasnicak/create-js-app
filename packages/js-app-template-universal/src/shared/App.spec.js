import React from 'react';
import renderer from 'react-test-renderer';

import App from './App';

test('renders App', () => {
  const component = renderer.create(<App />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
