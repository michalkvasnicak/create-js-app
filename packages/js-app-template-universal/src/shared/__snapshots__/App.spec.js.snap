exports[`test renders App 1`] = `
<div>
  <h1
    className={undefined}>
    Hello World
  </h1>
</div>
`;
